src/Static.ts

```diff
@@ -8,18 +8,37 @@ export class Static {
     }
 
     public static buildSectionBlock(mrkdwnValue: string): SectionBlock {
-        return { type: "section", text: { type: "mrkdwn", text: mrkdwnValue } };
+        return {type: "section", text: {type: "mrkdwn", text: mrkdwnValue}};
     }
 
     public static buildContextBlock(mrkdwnValue: string): ContextBlock {
-        return { type: "context", elements: [ { type: "mrkdwn", text: mrkdwnValue } ] };
+        return {type: "context", elements: [{type: "mrkdwn", text: mrkdwnValue}]};
     }
 
     public static buildSelectOption(text: string, value: string): Option {
-        return { text: this.buildTextElem(text), value: value };
+        return {text: this.buildTextElem(text), value: value};
     }
 
     public static buildTextElem(text: string): PlainTextElement {
-        return { type: "plain_text", text, emoji: true };
+        return {type: "plain_text", text, emoji: true};
+    }
+    public static buildVoteOptions(parameters: string [], start: number ): ActionsBlock[] {
+        const actionBlocks: ActionsBlock[] = [{ type: "actions", elements: [] }];
+        let actionBlockCount = 0;
+        // Construct all the buttons
+        for (let i = start; i < parameters.length; i++) {
+            if (i % 5 === 0 && i != 0) {
+                const newActionBlock: ActionsBlock = { type: "actions", elements: [] };
+                actionBlocks.push(newActionBlock);
```

Comment:
We should just make this one line if we're not going to re-use the defined constant again.